src/fsharp/XmlDoc.fs

```diff
@@ -180,11 +172,22 @@ type XmlDocCollector() =
         currentGrabPointCommentsCount <- currentGrabPointCommentsCount + 1
 
     member x.LinesBefore grabPointPos =
-        let lines = savedLinesAsArray.Force()
         match savedGrabPoints.TryGetValue grabPointPos with
-        | true, struct(startIndex, endIndex, _) -> lines.[startIndex .. endIndex]
+        | true, struct(startIndex, endIndex, _) ->
+            let linesBefore = Array.create (endIndex - startIndex + 1) ("", range0)
```

Comment:
Why not use a slice? Isn't this equivalent to `savedLines.[startIndex .. endIndex]`?

Reply:
`savedLines` is ResizeArray, and it doesn't support slicing :(

![image](https://user-images.githubusercontent.com/26364714/151538762-d91f24da-8324-40a5-b6f9-95fe83ad119d.png)
![image](https://user-images.githubusercontent.com/26364714/151538984-73a2e97b-a5df-4814-afb9-0d9f3e7f8750.png)

Reply:
Also, `List<T>` contains `List<T>.GetRange(Int32, Int32)` method, but it returns new `List<T>`, which is not very convenient for further processing

Reply:
I see, thanks